python/grass/jupyter/utils.py

```diff
@@ -47,8 +45,12 @@ def reproject_region(region, from_proj, to_proj):
     :return dict region: reprojected region as a dictionary with long key names
     """
     region = region.copy()
+    # reproject all corners, otherwise reproj. region may be underestimated
```

Comment:
Do you want to mention here the complete v.in.region based solution from r.import?